www/components/Input.js

```diff
@@ -54,14 +54,20 @@ const Input = ({
 
   //variant cases
   switch ( variant ) {
-    case 'default':
-      classNames += ' ' + styles.default;
+    case 'default':           //multi-conditional case statement
```

Comment:
For items with the `default` styles can we just have the bottom border become red? I think all you need for both conditions is something like

```
// switch statement you already have

if ( errorMessage ) {
  classNames += ' ' styles.error;
}
```

Where the `error` styles just set a new border color.

Reply:
@thudsonbu pushed the change